lib/school.rb

```diff
@@ -0,0 +1,21 @@
+class School
+    def initialize(time, hours)
+      @start_time = time
+      @hours = hours
+      @student_names = []
+    end
+    def add_student_name(student_name)
+        @student_names.push
+  end
```

Comment:
You are missing an end here!  So the syntax error is really halting you in your tracks.  Remember that all methods have a beginning (`def`) and an end `end`)